storage/Devices/Bcache.cc

```diff
@@ -125,9 +125,16 @@ namespace storage
 
 
     void
-    Bcache::attach_bcache_cset(BcacheCset* bcache_cset)
+    Bcache::add_bcache_cset(BcacheCset* bcache_cset)
```

Comment:
What is the motivation for this change? The term "attach" is used at may places in this concern, e.g. by the bcache tool (as seen in Bcache::Impl::do_attach_bcache_cset()).

Reply:
As discussed on IRC, this was changed for consistency with the rest of the public API.